ext/WaterLilyWriteVTKExt.jl

```diff
@@ -7,32 +7,33 @@ else
 end
 
 using WaterLily
+import WaterLily: vtkWriter, write!
 using Printf: @sprintf
 import Base: close
 
 """
-    vtk_grid(name;attrib,T)
+    vtkWriter(fname;attrib,dir,T)
 
 Generates a `vtkWriter` that hold the collection name to which the `vtk` files are written.
 The default attributes that are saved are the `Velocity` and the `Pressure` fields.
 Custom attributes can be passed as `Dict{String,Function}` to the `attrib` keyword.
 """
-struct vtkWriter
-    fname::String
-    dir_name :: String
-    collection::WriteVTK.CollectionFile
-    output_attrib::Dict{String,Function}
-    count::Vector{Int}
-    function vtkWriter(fname="WaterLily";attrib=default_attrib(),dir="vtk_data",T=Float32)
-        !isdir(dir) && mkdir(dir)
-        new(fname,dir,paraview_collection(fname),attrib,[0])
-    end
+struct VTKWriter
```

Comment:
@b-fg I had to rename the `struct` and remove the constructor function since it gave me an error when I had an empty function called `vtkWriter` in `WaterLily.jl` and I was defining a `struct` here with the same name.

Reply:
Yes, that is all good. Also in general types (and `struct`s) start with a capital letter.